.ahub/sam/exclude.txt

```diff
@@ -24,6 +24,7 @@
 
 # Test codes
 /ONE/tests
+/ONE/runtime/libs/misc/test
```

Comment:
@hseok-oh I dont need to add this according to https://github.com/Samsung/ONE/pull/8971#issuecomment-1103293004. Is it right? Then, I will remove this line.

Reply:
Right. Instead, we need to update `.ahub/tcchecker-tca/config.yaml`.